src/script/notification/NotificationRepository.test.ts

```diff
@@ -153,8 +153,13 @@ describe('NotificationRepository', () => {
         notification_content.trigger = trigger;
 
         if (_conversation.isGroup()) {
-          const titleLength = NotificationRepository.CONFIG.TITLE_LENGTH;
-          const titleText = `${_message.user().name()} in ${_conversation.display_name()}`;
+          const titleLength = NotificationRepository.CONFIG.TITLE_MAX_LENGTH;
```

Comment:
we can create function for the following block of code, since its used in several places in the file.

const titleLength = NotificationRepository.CONFIG.TITLE_MAX_LENGTH;
          const titleSectionLength = NotificationRepository.CONFIG.TITLE_LENGTH;

          const titleText = `${message.user().name()} in ${truncate(
            conversation.display_name(),
            titleSectionLength,
            false,
          )}`;

Reply:
good call